### PR TITLE
Voucher: Add min_usages parameter

### PR DESCRIPTION
**Pull request by raphaelm**
- [x] Tests

After merge

- [ ] pretixPOS support

### CODE REVIEW

File: src/pretix/base/models/event.py [hunk 1]
```diff
@@ -590,6 +590,7 @@ def set_defaults(self):
         self.settings.event_list_type = 'calendar'
         self.settings.invoice_email_attachment = True
         self.settings.name_scheme = 'given_family'
+        self.settings.payment_banktransfer_invoice_immediately = True
```
[the remark this answers is not in the document]
Comment (raphaelm) [reply src/pretix/base/models/event.py:593]: thanks, yes

[the remark this answers is not in the document]
Comment (raphaelm) [reply src/pretix/base/models/event.py]: I removed it

File: src/pretix/presale/templates/pretixpresale/event/voucher.html [hunk 2]
```diff
@@ -410,7 +410,16 @@ <h4 id="item-{{ item.pk }}-legend">{{ item.name }}</h4>
...
             {% if event.presale_is_running and options > 0 %}
                 <div class="row checkout-button-row">
-                    <div class="col-md-4 col-md-offset-8 col-xs-12">
+                    <div class="col-md-4 col-md-offset-8 col-xs-12 text-center">
+                        {% if voucher.min_usages > 1 %}
+                            <p class="text-muted">
+                                <small>
+                                    {% blocktrans trimmed with number=voucher.min_usages %}
+                                        You need to select at least {{ number }} products.
+                                    {% endblocktrans %}
```
Comment (pc-coholic) [inline src/pretix/presale/templates/pretixpresale/event/voucher.html:419]: This is a little bit untrue, as if the voucher has already been redeemed enough time, the min_usages do not apply anymore.

**Review by pc-coholic — COMMENTED** [review]
A little bit late to the party, sorry...

File: src/pretix/presale/templates/pretixpresale/event/voucher.html (hunk 2 above)
Comment (raphaelm) [reply src/pretix/presale/templates/pretixpresale/event/voucher.html:419]: thanks, fixed in 248e22c3d

**Pull request merged**